lms/djangoapps/discussion/tests/test_views.py

```diff
@@ -1463,7 +1463,11 @@ def test_forum_user_profile_xss_prevent(self, malicious_code, mock_threads, mock
         Test that XSS attack is prevented
         """
         mock_threads.return_value = [], 1, 1
-        mock_from_django_user.return_value.to_dict.return_value = {}
+        mock_from_django_user.return_value.to_dict.return_value = {
+            'upvoted_ids': [],
+            'downvoted_ids': [],
+            'subscribed_thread_ids': []
+        }
```

Comment:
This mock is the only thing I changed in my latest push; it needed to stub out a bit more data to complete the `_create_discussion_board_context(...)` call that is now a part of this codepath correctly.